source/server/configuration_impl.cc

```diff
@@ -87,6 +88,10 @@ void MainImpl::initialize(const envoy::config::bootstrap::v3::Bootstrap& bootstr
   // tracing configuration is missing from the bootstrap config.
   initializeTracers(bootstrap.tracing(), server);
 
+  // stats_config_ should be set before creating the ClusterManagers so that it is available
+  // from the ServerFactoryContext when creating the static clusters and stats sinks.
```

Comment:
what static clusters?  Bootstrap clusters?

Maybe just specifically say so that the lazy-init setting is available when creating clusters?